asreview/models/balance/double.py

```diff
@@ -49,6 +49,7 @@ class DoubleBalance(BaseBalance):
     """
 
     name = "double"
+    label = "Double"
```

Comment:
Can you add a bit more explanation like you do with the other labels? for example `Two way rebalancing function`